### DISCUSSION TITLE
copilot keymay problem in pycharm

### DISCUSSION THREAD

**Discussion by BeeFengs** [community/community#8028] [Copilot Conversations] [answered]
Hello,this is my second day using this plugin, I think this plugin will undoubtedly change the programming community, or even redefine the programming, when using this plugin, can study at a relatively low cost to obtain extremely wonderful experience, of course, there are some feeling can iterate experience, for example, when I use pycharm, after I wrote a comment, The shortcut key 'ALT+ [' cannot be used as a prompt to switch, because the shortcut key of different IDE software may cause the shortcut key conflict, so this shortcut key Settings need to consider how to switch the experience style, for example, can display some visual button mouse click selection, in addition, Will Copilot also be built into Juypter? I don't know if it's technically possible, but it sounds pretty good anyway, haha.

**Comment by jansorg** [comment, marked as answer]
@BeeFengs Thanks for your feedback! For now, you can assign your own shortcut at `Preferences > Key map > Plugin > GitHub Copilot`. Can you tell if the conflict is caused by a built-in or a 3rd-party extension? Which OS and keymap  are you using?

We'll think about a visual indicator.

Copilot is available in notebooks in the JetBrains IDEs supporting it. Were you looking for more advanced support?

**Reply by BeeFengs** [reply]
![image](https://user-images.githubusercontent.com/72483374/143212668-ef54c30c-845e-4302-af3f-3bec92d42624.png)
hello,I use windows 10,As shown in the picture, it is disabled, perhaps due to some version reasons.I think there needs to be a different experience path or method to provide this option. The first scenario should be in vscode and jetbrain tools, which need to ensure that they can be used without conflict.
